.prettierrc

```diff
@@ -16,8 +16,14 @@
         {
           "files": "*.sol",
           "options": {
-                "singleQuote": false 
-            }    
+                "singleQuote": false
+            }
+        },
+        {
+          "files": ["**/*.json", "**/*.yml"],
+          "options": {
+                "tabWidth": 2
+            }
         }
       ]
```

Comment:
```
  "lint-staged": {
    "*.{js,ts,sol,json}": [
      "prettier --write"
    ]
  },
```
Maybe not md (I think about the doc, to be checked)

Reply:
Thanks [Add json and yml extensions to lint-staged config](https://github.com/iExecBlockchainComputing/PoCo/pull/204/commits/4c19bef7c46d82427db2bc6055d1224b766e5115)